[error] examples/neg/typer/no-resume-in-object.effekt:5:22: Cannot find `resume`. Maybe you are trying to resume inside of an object literal and not as part of `try { ... } with ...`?
    def print(msg) = resume(())
                     ^
